Composer/packages/client/src/recoilModel/atoms/botState.ts

```diff
@@ -47,6 +47,32 @@ const emptyDialog: DialogInfo = {
   skills: [],
   isFormDialog: false,
 };
+
+const emptyLg: LgFile = {
+  id: '',
+  content: '',
+  diagnostics: [],
+  templates: [],
+  allTemplates: [],
+  imports: [],
+};
+
+type LgStateParams = { projectId: string; lgFileId: string };
+
+export const lgFileState = atomFamily<LgFile, LgStateParams>({
```

Comment:
This seems the same as lgFilesState.  Why make the name singular where lgFileIdsState is plural?